modules/core/src/main/java/org/apache/ignite/internal/IgnitionEx.java

```diff
@@ -1472,6 +1476,24 @@ private static void notifyStateChange(@Nullable String igniteInstanceName, Ignit
             lsnr.onStateChange(igniteInstanceName, state);
     }
 
+    /**
+     * Sets custom dependency resolver which provide override dependencies.
```

Comment:
"which provides overridden dependencies"
or
"which allows to override dependencies"